src/generatedSheetFunctions.js

```diff
@@ -1,4 +1,4 @@
-
+/* eslint-disable no-multi-spaces*/
```

Comment:
why?

Reply:
I find it weird that different files have a different set of disabled rules - it is confusing

Reply:
Because when you make the options object, you add spaces, if you don't do it, you'll have to disable another rule and that is when the options look like this: {currency: currency}

Reply:
`when you make the options object, you add spaces` doesn't mean much to mean

Reply:
https://github.com/santiment/google-spreadsheet-addon/blob/rework-for-v8/generators/getMetricGenerator.js#L54

Here, there are spaces. If those spaces went missing, another rule will come out and say, that there should be spaces after { and before }.

Reply:
what you link to is part of string, so rules should not apply to it

Reply:
It is a string, that gets generated into a file, the same file generator generates the rule disabling ontop of the file.

Reply:
So you're saying there are contradictory rules enabled by default? It would be very strange if that's true

Reply:
It is so in the case, when there is an automatically generated file, I need to write strange cases just so I can omit this contradiction. In the es5 lint config, there wasn't such rule, so we just went along with the {  }

Reply:
---
When you run `npm test` it shows something like this:
```
...
  234:82  error  Multiple spaces found before '}'  no-multi-spaces
  249:82  error  Multiple spaces found before '}'  no-multi-spaces
  264:79  error  Multiple spaces found before '}'  no-multi-spaces
  285:69  error  Multiple spaces found before '}'  no-multi-spaces

/Users/ivan/work/google-spreadsheet-addon/src/log.js
   7:3  warning  Unexpected console statement  no-console
  11:3  warning  Unexpected console statement  no-console
  18:5  warning  Unexpected console statement  no-console

✖ 34 problems (31 errors, 3 warnings)
  31 errors and 0 warnings potentially fixable with the `--fix` option.
```

So you can run `./node_modules/.bin/eslint --fix` and fix those things

Reply:
IMO the rule disabling should be allowed ATM. I will try to write the eslint --fix command into the package.json of the project, so it runs automatically after the file generation, but I will do it in another PR as I've tested out most of the functions in the webeditor and don't want to make new changes just about now. Thanks for the idea, I think it will work